src/Task/ComposerNormalize.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+namespace GrumPHP\Task;
+
+use GrumPHP\Runner\TaskResult;
+use GrumPHP\Task\AbstractExternalTask;
+use GrumPHP\Task\Context\ContextInterface;
+use GrumPHP\Task\Context\GitPreCommitContext;
+use GrumPHP\Task\Context\RunContext;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+/**
+ * Class ComposerNormalize.
+ */
+class ComposerNormalize extends AbstractExternalTask
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function canRunInContext(ContextInterface $context)
+    {
+        return $context instanceof GitPreCommitContext || $context instanceof RunContext;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getConfigurableOptions()
+    {
+        $resolver = new OptionsResolver();
+        $resolver->setDefaults([
+            'indent_size' => 4,
+            'indent_style' => 'space',
+            'update_lock' => true,
+            'verbose' => false,
+        ]);
+
+        $resolver->addAllowedValues('indent_style', ['tab', 'space']);
+        $resolver->addAllowedTypes('indent_size', ['int']);
+        $resolver->addAllowedTypes('update_lock', ['bool']);
+        $resolver->addAllowedTypes('verbose', ['bool']);
+
+        return $resolver;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getName()
+    {
+        return 'composer_normalize';
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function run(ContextInterface $context)
+    {
+        $config = $this->getConfiguration();
+        $files = $context->getFiles()
+          ->path(pathinfo('composer.json', PATHINFO_DIRNAME))
+          ->name(pathinfo('composer.json', PATHINFO_BASENAME));
```

Comment:
Indentation appears to be off by 2 here (and in a few other places).

Perhaps run

```
$ composer require friendsofphp/php-cs-fixer
```

followed by

```
$  php-cs-fixer fix --rules=@PSR2 src/Task/ComposerNormalize.php
```

and while we are it, also 

```
$  php-cs-fixer fix --rules=@PSR2 spec/Task/ComposerNormalizeSpec.php
```

?

Reply:
Fixed.